core/src/datayoga_core/producer.py

```diff
@@ -13,7 +13,7 @@ def __init__(self, msg_id: str, value: Dict[str, Any]):
 class Producer(Block):
 
     @abstractmethod
-    async def produce(self) -> AsyncGenerator[List[Message], None]:
+    def produce(self) -> AsyncGenerator[List[Message], None]:
         """Produces data
 
         Returns:
```

Comment:
not updated return type:
```
Generator[Message]: Produced data
```

Should be:
```
AsyncGenerator[List[Message], None]
```

Or even,
```
        Yields:
            AsyncGenerator[List[Message], None]: A generator of message batches.
```

Reply:
✅